scripts/docker_entrypoint.sh

```diff
@@ -0,0 +1,125 @@
+#!/usr/bin/env bash
+set -eo pipefail # stop script execution if any errors are encountered
+
+# script help info
+function usage() {
+    cat <<USAGE
+
+    Usage: 
+        $0 build -b binary_path -c config_path [KB path]
+        $0 run -b binary_path -c config_path [sc-machine args]
+
+    Options:
+        build <PATH>:       Rebuilds KB from sources (provide absolute path to the source folder or repo.path file).
+        run <args>:         Starts sc-machine. Arguments passed to this command will be redirected to sc-machine binary. You can set sc-server options from common config.
+
+        Setting REBUILD_KB environment variable inside the container will trigger a KB rebuild. Setting custom starting point for sc-builder can be done using KB_PATH environment variable, "/kb" is used as a default KB_PATH.
+        CONFIG_PATH and BINARY_PATH environment variables can provide the respective settings if the use of flags is undesirable.
```

Comment:
they are lowercase a couple of lines above

Reply:
This docker entrypoint copy pasted from sc-machine

Reply:
binary_path is not the same that is BINARY_PATH